gamemodes/oresgyn/gamemode/rounds/sv_rounds.lua

```diff
@@ -76,6 +77,10 @@ function endRound(winner)
     end)
 end
 
+net.Receive(NET_PLAYER_JOIN, function(len, ply)
+    sendPlayerCurrentRoundStatus(ply)
+end)
+
```

Comment:
Sending the current round status to a new player didn't work properly.
Now it does.